operatorcourier/validate.py

```diff
@@ -85,7 +85,7 @@ def _bundle_validation(self, bundle, repository=None):
                 validationDict[self.pkgsKey] = False
 
         valid = True
-        for key, value in validationDict.items():
+        for _, value in validationDict.items():
```

Comment:
While this seems like a valid thing to do, it seems unrelated to the purpose of this pr. Can you please remove it and create a separate pr for it? Or at least a separate commit and explain why this change is being made in a separate commit message?

Reply:
Sure, thanks.

Reply:
```python
for value in validationDict.values():
```